src/components/FeatureViews/Registration/RegistrationCart/StudentRegistrationsEntry.js

```diff
@@ -67,8 +67,8 @@ function RegistrationEntry({registration: {course, numSessions, checked}, studen
 						{course.academicLevelPretty}
 						{separator()}
 						{moment(course.startDate, ["YYYY-MM-DD"]).format("ddd")} {' '}
-						{moment(course.startTime, ["HH.mm"]).format("h:mm A")}{" - "}
-						{moment(course.endTime, ["HH.mm"]).format("h:mm A")}
+						{moment(course.availabilityList[0].startTime, ["HH.mm"]).format("h:mm A")}{" - "}
```

Comment:
this is only listing one of the availabilities. we need to create a component that would display all of the availabilities